scripts/update-repo-internal.sh

```diff
@@ -16,17 +16,20 @@ RELEASE="${DISTRO##*-}"
 case $PARENT in
     centos)
         mkdir -p repo/$PARENT/$RELEASE/packages
-        find package/$DISTRO/ -name *.rpm | \
+        find package/$DISTRO/ -name i'*.rpm' | grep -v lstore-release.rpm | \
```

Comment:
Ah, no prob. I didn't know if there was some secret 'find' arg I was missing. I'll tidy it up in a sec (need to run to the store)